exercises/practice/ari/.docs/instructions.md

```diff
@@ -0,0 +1,102 @@
+# Instructions
+
+As we grow up, we can understand more and more complex texts. But how to assess the level of complexity of a given text, and how to teach the computer to do so? In this exercise, you will write a program determining how difficult a text is and for what age it is best suited.
```

Comment:
Long lines should be wrapped. Either wrap at a fixed column or split into one sentence per line.

Reply:
done